eva_submission/xlsx/xlsx_validation.py

```diff
@@ -89,20 +89,33 @@ def semantic_validation(self):
             elif len(accessions) > 1:
                 self.error_list.append(f'In Analysis, Reference {reference} resolve to more than one accession: {accessions}')
 
+        correct_taxid_sc_name = {}
         # Check taxonomy scientific name pair
         taxid_and_species_list = set([(row['Tax Id'], row['Scientific Name']) for row in self.metadata['Sample'] if row['Tax Id']])
         for taxid, species in taxid_and_species_list:
             try:
                 scientific_name = get_scientific_name_from_ensembl(int(taxid))
                 if species != scientific_name:
```

Comment:
The idea is to only allow correction if the difference is negligible 
```suggestion
                if species != scientific_name and species.lower() == scientific_name.lower():
```

Then to add an 
```python
elif species != scientific_name 
    self.error_list.append(f'In Samples, Taxonomy {taxid} and scientific name {species} are inconsistent')
```

Reply:
done